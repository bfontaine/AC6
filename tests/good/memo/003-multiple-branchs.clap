val c = case {
  | A[x] => x
  | B[x] => x + 1
  | C[x] => x + 2
}
val xA = c ( A[0] )
val xB = c ( B[0] )
val xC = c ( C[0] )
