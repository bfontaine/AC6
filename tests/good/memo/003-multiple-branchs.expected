:- c = <code>
:- xA = 0
:- xB = 1
:- xC = 2
